packetbeat/protos/rpc/nfs.go

```diff
@@ -0,0 +1,42 @@
+package rpc
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+	"time"
+)
+
+type Nfs struct {
+	xdr   Xdr
+	vers  uint32
+	proc  uint32
+	event *common.MapStr
```

Comment:
I would recommend not using a pointer here. Maps are reference types so they are always passed by reference.